supertokens_python/recipe/session/with_jwt/recipe_implementation.py

```diff
@@ -177,11 +181,51 @@ async def update_access_token_payload(
         )
 
         return await og_update_access_token_payload(
-            session_handle, new_access_token_payload, user_context
+            session_information.session_handle, new_access_token_payload, user_context
+        )
+
+    async def update_access_token_payload(
+        session_handle: str,
+        new_access_token_payload: Dict[str, Any],
+        user_context: Dict[str, Any],
+    ) -> bool:
+        # TODO: Node SDK allows non dict values of new_access_token_payload. But what should we do here?
+
+        session_information = await original_implementation.get_session_information(
+            session_handle, user_context
+        )
+        assert session_information is not None  # TODO: Is this a valid assumption?
```

Comment:
You should not assert it. Check if it is defined, and if not, return false. (This also needs to be done in the node SDK, and have commented this)

Reply:
done.